app/src/components/inputs/autocompletes/common/autocompleteMenu/autocompleteMenu.jsx

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable react/no-unknown-property */
```

Comment:
Please apply this exception near the place where it is used and describe with comment.